ui/package.json

```diff
@@ -25,17 +25,7 @@
     "typedoc": "typedoc"
   },
   "workspaces": [
-    "app",
-    "components",
-    "core",
-    "dashboards",
-    "explore",
-    "e2e",
-    "internal-utils",
-    "panels-plugin",
-    "plugin-system",
-    "prometheus-plugin",
-    "storybook"
+    "*"
```

Comment:
perhaps the build of the UI is failing because of this change ?

Reply:
Maybe, I also didn't include the update package-lock.json. Might also be that...

> '@perses-dev/tempo-plugin@0.41.1' is not in this registry.

Reply:
Lemme try a few things and push.

Reply:
Good luck ! https://www.youtube.com/watch?v=1cEy4UyYHI0 maybe it can help ;)